spec/actors/hyrax/actors/file_actor_spec.rb

```diff
@@ -173,7 +173,7 @@ class FileSetWithExtras < FileSet
       end
     end
 
-    it 'uses the provided mime_type' do
+    xit 'uses the provided mime_type' do
```

Comment:
This pending test passes for CCI, but was failing locally for several devs.  Since the code this is testing is not complete and the test is pending, just fully skip it with xit.